M/MicrosoftMPI/build_tarballs.jl

```diff
@@ -28,6 +28,7 @@ fi
 cd ${WORKSPACE}/destdir/
 
 chmod +x *.exe
+chmod +x *.dll
```

Comment:
No need to, now do this automatically: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/992

Reply:
Just do a dummy edit to trigger registration

Reply:
done!